### PR TITLE
Merge in Mark's stuff

### PR DESCRIPTION
**Pull request by fyellin**
Mark has added more cspyce functions.  He also automated the generation of the cspyce0_info.py files from the .i files.  

This is his changes, with some slight cleanup by me

### CODE REVIEW

File: cspyce/cspyce1.py [hunk 1]
```diff
@@ -848,6 +1019,7 @@ def assign_docstring(func, note=""):
 for name in CSPYCE_SIGNATURES:
     func = globals()[name]
     func.ABSTRACT  = CSPYCE_ABSTRACT[name]
+    func.ABSTRACT  = CSPYCE_ABSTRACT[name]
```
Comment (rfrenchseti) [inline cspyce/cspyce1.py:1022]: Duplicate line?

**Review by rfrenchseti — CHANGES_REQUESTED** [review]
There is an empty file called "2". Is this supposed to be there?

File: cspyce/cspyce1.py [hunk 2]
```diff
@@ -4,8 +4,16 @@
 # module cspyce.cspyce1
 #
 # This module provides several enhancements over the low-level cspyce0 interface
-# to the CSPICE library. See cspyce/cspyce0/__init__.py for more information
-# about cspyce0.
+# to the CSPICE library.
+#
+# The cspyce0 module contains Python interfaces to all CSPICE functions likely
```
[the remark this answers is not in the document]
Comment (fyellin) [reply cspyce/cspyce1.py:9]: Fixed

File: cspyce/cspyce1.py [hunk 3]
```diff
@@ -29,37 +37,37 @@
...
-#
-# In the cspyce1 module, we introduce two new options to function erract, which
-# is used in CSPICE to control the error handling options. The options are
-# 'EXCEPTION' and 'RUNTIME'. When error handling is set to one of these values,
-# Python exception handling takes over. Error conditions raise exceptions
-# rather than changing the value of failed(). The difference is that 'RUNTIME'
-# will consistently raise a RuntimeError exception, whereas 'EXCEPTION' will
-# tailor the type of exception to the situation.
+# Python's exception handling mechanism obviates the need for this approach. In
+# cspyce1, all CSPYCE runtime errors raise Python exceptions. You should never
```
[the remark this answers is not in the document]
Comment (fyellin) [reply cspyce/cspyce1.py:45]: Mark's comment.  But I agree with you

File: cspyce/cspyce1.py (hunk 1 above)
Comment (fyellin) [reply cspyce/cspyce1.py:1022]: Fixed

**Comment by fyellin** [comment]
I think I made all changes this time.

**Pull request merged**